src/loader.js

```diff
@@ -115,6 +118,39 @@ function setupModuleLoader(window) {
           _runBlocks: runBlocks,
 
           /**
+           * @ngdoc method
+           * @name angular.Module#info
+           * @module ng
+           *
+           * @param {Object=} info Information about the module
+           * @returns {Object|Module} The current info object for this module if called as a getter,
+           *                          or `this` if called as a setter.
+           *
+           * @description
+           * Additional info about this module
+           * For example you could put the version of the module in here.
+           *
+           * ```js
+           * angular.module('myModule', []).info({ version: '1.0.0' });
+           * ```
+           *
+           * You can retrieve this information during runtime via the
+           * {@link $injector#modules `$injector.modules`} property:
+           *
+           * ```js
+           * var version = $injector.modules['myModule'].info().version;
```

Comment:
I mean use it in place of `$injector.modules['myModule'].info()` - i.e. at "runtime" - so it should have been added to the injector.
I was mainly curious basically. `$injector.modules['myModule']` works for me :smiley:

--
Hm...since we brought it up, `$injector` is kind of ambiguous, since it may refer to the `providerInjector` (which doesn't know about `modules`) or the `instanceInjector` (which does).

I doubt anyone is using the `providerInjector`, but still it might be a good idea to add `modules` on the `providerInjector` as well (for symmetry).